components/alibi-detect-server/adserver/server.py

```diff
@@ -233,6 +233,13 @@ def post(self):
         response = self.model.process_event(request, headers)
         seldon_response = SeldonResponse.create(response)
 
+        runtime_metrics = seldon_response.metrics
+        if runtime_metrics is not None:
+            if validate_metrics(runtime_metrics):
+                self.seldon_metrics.update(runtime_metrics, "ce_server")
```

Comment:
is ce_server not a bit generic. What is this argument used for?

Reply:
This is an extra tag, more specifically it's the "method" tag, which in the case of the metrics in the python wrapper it would be "predict", "feedback", "input_transform", etc. In this case it would just be the "ce_server" metrics. I guess we could have it more specific to the type of server, such as DRIFT / OUTLIER / CUSTOM_METRICS?